frame/collective/src/lib.rs

```diff
@@ -751,7 +767,10 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 		if approve {
 			if position_yes.is_none() {
-				voting.ayes.push(who.clone());
+				voting
+					.ayes
+					.try_push(who.clone())
+					.expect("Proposal voting ayes can't overflow; qed");
```

Comment:
Rather return an error; there is no reason to panic the runtime if this does not hold.